### PR TITLE
LPS-129739 FFMPEG backed audio processor

### PR DESCRIPTION
**Pull request by adolfopa**
Upload a supported audio file to Documents & Media. After a while, you should get an audio player in the document details. That will play a, potentially, lower quality version of the original file, encoded as an mp3 or ogg audio file.There are integration tests to verify that the conversion works.

### CODE REVIEW

**Comment by adolfopa** [comment]
ci:test:sf

**Comment by adolfopa** [comment]
ci:test:relevant

**Review by AliciaGarciaGarcia — APPROVED** [review]

**Comment by adolfopa** [comment]
ci:forward

File: modules/apps/document-library/document-library-preview-audio-test/src/testIntegration/java/com/liferay/document/library/preview/audio/test/DLAudioFFMPEGAudioConverterTest.java [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+		UserTestUtil.setUser(TestPropsValues.getUser());
+
+		_group = GroupTestUtil.addGroup();
+
+		_serviceContext = ServiceContextTestUtil.getServiceContext(
+			_group.getGroupId(), 0);
+	}
+
+	@Test
+	public void testDoesNotGenerateAudioPreview() throws Exception {
```
Comment (AliciaGarciaGarcia) [inline modules/apps/document-library/document-library-preview-audio-test/src/testIntegration/java/com/liferay/document/library/preview/audio/test/DLAudioFFMPEGAudioConverterTest.java:73]: @adolfopa 
FYI : This test  is failing if the server has the ffmpeg audio converter enabled.

```

com.liferay.document.library.preview.audio.test.DLAudioFFMPEGAudioConverterTest > testDoesNotGenerateAudioPreview FAILED
    java.lang.AssertionError
```

Comment (adolfopa) [reply modules/apps/document-library/document-library-preview-audio-test/src/testIntegration/java/com/liferay/document/library/preview/audio/test/DLAudioFFMPEGAudioConverterTest.java:73]: Good catch @AliciaGarciaGarcia. I'll change that tomorrow :+1:

**Pull request closed without merging**